{
  "cve_id": "CVE-2023-26150",
  "timestamp": "2025-01-14 00:42:36",
  "phases": {
    "primary": {
      "completed": true,
      "timestamp": "2025-01-10 10:39:56"
    },
    "secondary": {
      "completed": true,
      "timestamp": "2025-01-11 20:55:58"
    },
    "extraction": {
      "completed": true,
      "timestamp": "2025-01-13 09:11:56"
    }
  },
  "urls": [
    {
      "url": "https://gist.github.com/artfire52/84f7279a4119d6f90381ac49d7121121",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-10 10:39:06"
    },
    {
      "url": "https://github.com/FreeOpcUa/opcua-asyncio/commit/2be7ce80df05de8d6c6ae1ebce6fa2bb7147844a",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-10 10:39:06"
    },
    {
      "url": "https://github.com/FreeOpcUa/opcua-asyncio/commit/b4106dfd5037423c9d1810b48a97296b59cde513",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-10 10:39:06"
    },
    {
      "url": "https://github.com/FreeOpcUa/opcua-asyncio/issues/1014",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-10 10:39:06"
    },
    {
      "url": "https://github.com/FreeOpcUa/opcua-asyncio/pull/1015",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-10 10:39:06"
    },
    {
      "url": "https://github.com/FreeOpcUa/opcua-asyncio/releases/tag/v0.9.96",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-10 10:39:06"
    },
    {
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-ASYNCUA-5673435",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-10 10:39:06"
    }
  ]
}